presto-record-decoder/pom.xml

```diff
@@ -51,6 +51,19 @@
             <artifactId>joda-time</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.avro</groupId>
```

Comment:
We put versions in the top level pom.

Reply:
It might be appropriate to do here if we really need a different version of Avro between the Hive and Kafka connectors, though it's preferred to use the `dependencyManagement` version in the root POM.